test/events/page.test.js

```diff
@@ -25,7 +25,7 @@ describe('page', function () {
 
 	after(function () {
 		new Queue('requests').replace([]); // Empty the queue as PhantomJS doesn't always start fresh.
-		destroy('config'); // Empty settings.
```

Comment:
This was a bug that never got hit in the previous test runner because this was always the last test to be executed.

The bug is that `destroy` will delete the config property when what we want to do is reset the config property to it's default value, which is an empty object.

Our new test runner hit this bug, which is why it was spotted now, I'll backport this to the current version of o-tracking 👍